packages/info/src/index.ts

```diff
@@ -38,10 +38,7 @@ export default async function info(...args): Promise<string[]> {
     const infoArgs = parsedArgs.opts;
     const envinfoConfig = {};
 
-    if (parsedArgs.unknownArgs.length > 0) {
-        logger.error(`Unknown argument: ${parsedArgs.unknownArgs}`);
-        process.exit(2);
-    }
+    handleUnknownArgs(parsedArgs.unknownArgs);
```

Comment:
We should avoid handling this in sub packages, unknown args and commands should be handled in `webpack-cli`

Reply:
https://github.com/webpack/webpack-cli/pull/1703#discussion_r461366612

Reply:
So we need to refactor our commands, and register their command names and flags before parsing args

Reply:
We can do it separately